app/views/todos/index.html.erb

```diff
@@ -6,10 +6,13 @@
 
   <% if @all_todos.any? %>
     <section class="main">
-      <input class="toggle-all" type="checkbox">
-      <label data-controller="todos" data-action="click->todos#toggleAll">
-        Mark all as complete
-      </label>
+      <%= form_with url: bulk_update_todos_path, html: { data: { controller: "todos", target: "todos.form" } } do %>
+        <input class="toggle-all" type="checkbox">
+        <label data-action="click->todos#toggleAll">
+          Mark all as complete
+        </label>
+      <% end %>
+
```

Comment:
I'm curious if we can make that `label` a submit button. If so, we might be able to lean entirely on ujs i.e. `remote: true` on the form. That would allow us to simplify further and eliminate the Stimulus controller.

Reply:
I thought about that but thought it would mess up the styling. I’ll give it a try though.

Reply:
Yeah. I'd rather deal with the styling changes I think as we're trying to illustrate how to do this with the least moving parts possible.

Reply:
also - `form_with` has `remote: true` by default.